src/libs/intents/localMethods.spec.ts

```diff
@@ -45,9 +48,9 @@ describe('asyncLogout', () => {
 
 describe('backToHome', () => {
   it('should handle Navigation', async () => {
-    const client = {}
+    const client = {} as unknown as CozyClient
 
-    localMethods(client).backToHome()
+    await (localMethods(client) as NativeMethodsRegister).backToHome()
```

Comment:
Why do we need to specify `as NativeMethodsRegister` here? There seems to be no ambiguity from the import.

Reply:
Edit: ok I just saw the `CustomMethods` thing